.github/workflows/ci-cppcheck.yml

```
@@ -13,7 +13,7 @@ jobs:
   build:
 
     runs-on: ci-cppcheck
-    if: ${{ github.repository != 'intel/pcm' }}        
+    if: ${{ github.repository != 'khulnasoft/pcm' }}        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)